src/api-reference/request/request.markdown

```diff
@@ -1,244 +1,411 @@
-`CurrencyCode`	|	`string`	|	-	|	The 3-letter ISO 4217 currency code for the Request currency. The Request currency is defined as the Request creator's default reimbursement currency.
-`CurrencyName`	|	`string`	|	-	|	The currency name for the Request currency. The Request currency is defined as the Request creator's default reimbursement currency.
-`Custom1 through Custom20`	|	`CustomField`	|	-	|	The details from the Custom fields. These fields may not have data, depending on the configuration.
-`EmployeeName`	|	`string`	|	-	|	The first, middle (or middle initial), and last name of the employee who created the Request.
-`EndDate`	|	`string`	|	-	|	The end date of the Request.
-`EndTime`	|	`string`	|	-	|	The end time for the Request.
-`Entries`	|	`Array`	|	[Entry](#entry)	|	This parent element has a RequestEntry child element for each entry. See the RequestEntry model for the full list of child elements.
-`EverSentBack`	|	`string`	|	-	|	Indicates whether the Request has ever been sent back to the employee. Format: Y/N
-`Exceptions`	|	`Array`	|	[Exception](#exception)	|	This parent element has an Exception child element for each exception. See the Exception model for the full list of child elements.
-`ExtensionOf`	|	`string`	|	-	|	The ID of the initial Request that this Request is an extension of or adendum to.
-`HasException`	|	`string`	|	-	|	Indicates whether the Request has exceptions. Format: Y/N
-`HeaderFormID`	|	`string`	|	-	|	The unique identifier of the Header Form resource (See the "Forms" resource for more information).
-`LastModifiedDate`	|	`DateTime`	|	-	|	The date the Request was last modified. Format: YYYY-MM-DDThh:mm:ss
-`LoginID`	|	`string`	|	-	|	The Concur login ID of the Request owner.
-`Name`	|	`string`	|	-	|	The name of the Request.
-`PolicyID`	|	`string`	|	-	|	The unique identifier of the policy that applies to this Request. Maximum length 64 characters.
-`PolicyName`	|	`string`	|	-	|	The name of the policy that applies to this Request.
-`Purpose`	|	`string`	|	-	|	The purpose of the Request.
-`RequestID`	|	`string`	|	-	|	The unique key for the Request.
-`StartDate`	|	`string`	|	-	|	The start date of the Request.
-`StartTime`	|	`string`	|	-	|	The start time for the Request.
-`SubmitDate`	|	`DateTime`	|	-	|	The date the Request was submitted. Format: YYYY-MM-DDThh:mm:ss.
-`TotalApprovedAmount`	|	`string`	|	-	|	The total amount of approved expenses in the Request.
-`TotalPostedAmount`	|	`string`	|	-	|	The total amount of the Request.
-`TotalRemainingAmount`	|	`string`	|	-	|	The total amount of expenses remaining in the Request.
-`UserPermissions`	|	`UserPermissions`	|	-	|	The actions that the user is allowed to perform on the Request.
-
+PUT /v4/expenses/{expenseUuid}
 
+*Parameters:*
+Name | Parameter Type | Data Type | Description
+--- | :---: | :---: | ------
+expenseUuid | path | string | __Required__ The unique identifier of the Expense to update
+body | object | [Expense](#expense) | Required The expense content to update
```

Comment:
Name : -
Parameter Type = body
Data Type = [Expense](#expense)
Description = **Required** The expense content to update